kato/kato-cf/src/main/groovy/com/netflix/spinnaker/kato/cf/deploy/converters/CloudFoundryDeployAtomicOperationConverter.groovy

```diff
@@ -42,8 +42,22 @@ class CloudFoundryDeployAtomicOperationConverter extends AbstractAtomicOperation
     if (input.containsKey('loadBalancers') && input.loadBalancers instanceof List) {
       input.loadBalancers = input.loadBalancers.join(',')
     }
+
     def converted = objectMapper.convertValue(input, CloudFoundryDeployDescription)
+
     converted.credentials = getCredentialsObject(input.credentials as String)
+
+    /**
+     * Convert supported template options, e.g. s3://example.com/{job}-{buildNumber}/
```

Comment:
In the aws credentials config I'm using `{{ }}` (mustache style) as the placeholder, it might be worth following that for consistency in the config files in a multi-cloudProvider deployment